jetbrains-core/src/software/aws/toolkits/jetbrains/settings/AwsSettingsConfigurable.java

```diff
@@ -84,6 +93,9 @@ private void createUIComponents() {
         samHelp = createHelpLink("lambda.sam.cli.install_url");
         samHelp.setEnabled(false);
         samExecutablePath = createCliConfigurationElement(getSamExecutableInstance(), SAM);
+        cfnLintHelp = createHelpLink("cloudformation.linter.install_url");
+        cfnLintHelp.setEnabled(false);
```

Comment:
Doing this disables the link. I am not sure why the other help links are disabled as well. 

![image](https://user-images.githubusercontent.com/5374887/80936079-6d77f900-8d84-11ea-987d-146042d70970.png)

If I set this to `true` they appear as proper links.

![image](https://user-images.githubusercontent.com/5374887/80935999-21c54f80-8d84-11ea-98b2-7890f312db1b.png)

Reply:
Welp. That's bad.